CryptoTechReminderSystem.Main/.env-default

```
@@ -2,4 +2,7 @@ SLACK_TOKEN=
 HARVEST_TOKEN=
 HARVEST_ACCOUNT_ID=
 HARVEST_USER_AGENT=
-LEARNTECH_CHANNEL_ID=
\ No newline at end of file
+LEARNTECH_CHANNEL_ID=
```

```suggestion
SLACK_LEARNTECH_CHANNEL_ID=
```
Since we've been naming all the other environment variables beginning with the source?

---

Could make it more generic and use `SLACK_CHANNEL_ID`.

---

Resolved here https://github.com/chubberlisk/crypto_tech/pull/31/commits/75edcdbf331cea21af64eb290e3d616733d02407.